.spi.yml

```diff
@@ -0,0 +1,4 @@
+version: 1
+builder:
+  configs:
+    - documentation_targets: [Tracing]
```

Comment:
I'm not very familiar with the Swift package index yet and its DocC support, but would this also surface the documentation from the `Instrumentation` target given that `Tracing` depends on it, or would we want to declare both as `documentation_targets` here?

Reply:
I guess I am not sure either. Let me include it just in case.